_notices/rsn0034.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 34 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation announcement for Pascal GPU support in Release v23.12"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.12"
+notice_created: 2023-10-04
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-10-04
+---
+
+## Overview
+
+RAPIDS team is planning to deprecate Pascal GPUs support in its `v23.12` release and remove support in its `v24.02` release
```

Comment:
```suggestion
RAPIDS team is planning to deprecate support for Pascal GPUs in its `v23.12` release and to remove support in its `v24.02` release. `v23.12` will be the last version of RAPIDS to support Pascal GPUs.
```

Reply:
Should we mention something like “Compute Capability 7.0 or higher will be required”?

Reply:
Good idea. I think that fits better in the "impact" section below, so have added there.